server/plat/rtcache.cpp

```diff
@@ -19,74 +19,75 @@
 
 #include "rtcache.hpp"
 
-std::string rtAppStatusCache::Netflix_AppCacheId = "DialNetflix";
-std::string rtAppStatusCache::Youtube_AppCacheId = "DialYoutube";
-
-std::string rtAppStatusCache :: getAppCacheId(const char *app_name)
-{
-     printf("RTCACHE : %s\n",__FUNCTION__);
-
-     if(!strcmp(app_name,"Netflix"))
-          return rtAppStatusCache::Netflix_AppCacheId;
-     else if(!strcmp(app_name,"YouTube"))
-          return rtAppStatusCache::Youtube_AppCacheId;
-     else
-     {
-          printf("Invalid App Name\n");
-          return "INVALID";
-     }
-
-}
-
-void rtAppStatusCache :: setAppCacheId(const char *app_name,std::string id)
-{
-     printf("RTCACHE : %s\n",__FUNCTION__);
-     if(!strcmp(app_name,"Netflix"))
-     {
-         rtAppStatusCache::Netflix_AppCacheId = id;
-         printf("App cache Id of Netflix updated to %s\n",rtAppStatusCache::Netflix_AppCacheId.c_str());
-     }
-     else if(!strcmp(app_name,"YouTube"))
-     {
-         rtAppStatusCache::Youtube_AppCacheId = id;
-         printf("App cache Id of Youtube updated to %s\n",rtAppStatusCache::Youtube_AppCacheId.c_str());
-     }
-     else
-     {
-         printf("Invalid App Name \n");
-     }
-
-}
+#define printf(...) printf(__VA_ARGS__); \
+                    fflush(stdout);
```

Comment:
I understand there is no logging facility in xdial, but maybe some macro would be more in place (instead of overriding printf like this)

Reply:
(there seem to be some debugging utils in server/gdial-debug.h)

Reply:
Used g_print